src/test/emulators/auth/helpers.ts

```diff
@@ -113,6 +114,29 @@ export function registerAnonUser(
     });
 }
 
+export function registerMfaUser(
```

Comment:
Combine this with `registerUser` above. afacit this is just `registerUser` but with an additional request field.